cli/offchain-commands/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "partner-chains-offchain-commands"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = { workspace = true }
+clap = { workspace = true }
+ed25519 = { workspace = true, features = ["alloc"]}
+ed25519-zebra  = { workspace = true}
+hex = { workspace = true }
+hex-literal = { workspace = true }
+plutus = { workspace = true, default-features = true }
+plutus-datum-derive = { workspace = true, default-features = true }
+secp256k1 = { workspace = true, features = ["std", "global-context"] }
+sidechain-domain = { workspace = true, default-features = true }
+sp-io = { workspace = true, default-features = true }
+thiserror = { workspace = true }
+serde = { workspace = true }
+serde_json = { workspace = true }
```

Comment:
I would prefer to add deps when they are needed. This way we can easily end up with some extra ones